charts/example-configurations/minikube-values-renkulab-template.yaml

```diff
@@ -1,113 +1,37 @@
-#
+# against the gitlab of another Renku deployment. We also need to configure the keycloak instance which is part
+# of Renku to use Renku deployment as identity provider. See the developer docs
+# (https://renku.readthedocs.io/en/latest/developer/example-configurations/renkulab.html)
+# for instructions on how to use this values file.
 
-ingress:
-  enabled: true
-  annotations:
-    kubernetes.io/ingress.class: 'nginx'
-    nginx.ingress.kubernetes.io/ssl-redirect: 'false'
-    nginx.ingress.kubernetes.io/proxy-body-size: '0' # Adjust to a reasonable value for production to avoid DOS attacks.
-    nginx.ingress.kubernetes.io/proxy-request-buffering: 'off' # Needed if GitLab is behind this ingress
-  hosts:
-  - ""
-
-global:
-  development:
-    gitlabUrl: &renku-gitlab-url # Put the gitlab URL here!
-    gitlabClientId: &gitlab-client-id # Put Application Id here!
-    gitlabClientSecret: &gitlab-client-secret # Put Application Secret here!
-  gitlab:
-    clientSecret: e3ecd740-0892-4610-ad69-c7f4886f4533
-    postgresPassword: 3b0aa4bae4fbbe167143f583e35eda18
-    sudoToken: 214981ee49479ecbee56359b70ad25dc87edd0e9ea569b7b8c522c3bf979446a
-    urlPrefix: /gitlab
-  keycloak:
-    postgresPassword: 49d0aa1b8a1095415cc975d766933e7b
-  jupyterhub:
-    postgresPassword: d6f59c5df645abe2fd1a0572b6bd5a8d
-  gateway:
-    clientSecret: 259a72d8-2c40-435f-9e0f-24df7c1a7dbc
-    gitlabClientSecret: 4e35410e4116d2280e29c6da6aaaa89fc1ab77364c6352aeb5adbd037d49590a
-  renku:
-    version: 'latest'
-  graph:
-    dbEventLog:
-      postgresPassword: 58dd574b6a19f9b6b2cedaf20f402abf
-    tokenRepository:
-      postgresPassword: 12db5d7b3f0e38c79ad08b6e7f2bcaf4
+variables:
+  gitlabUrl:          &renkuGitlabURL      # Put the gitlab URL here!
+  gitlabClientId:     &gitlabClientId      # Put Application Id here!
+  gitlabClientSecret: &gitlabClientSecret  # Put Application Secret here!
 
 gitlab:
   enabled: false
 
 notebooks:
   gitlab:
-    url: *renku-gitlab-url
+    url: *renkuGitlabURL
     registry:
-      host: # registry host, e.g. registry.renkulab.io
-  securityContext:
-    enabled: false
+      host: # Put the registry host here! e.g. registry.renkulab.io
   jupyterhub:
-    rbac:
-      enabled: true
     hub:
-      cookieSecret: 7c232b97bda321be7e59b9834dfadbb89652da1e0150dff9b3dadb9aa8b4caab
-      baseUrl: '/jupyterhub/'
-      db:
-        type: postgres
-        url: postgres+psycopg2://jupyterhub:d6f59c5df645abe2fd1a0572b6bd5a8d@renku-postgresql:5432/jupyterhub
-      services:
-        notebooks:
-          apiToken: notebookstoken
-        gateway:
-          admin: true
-          oauth_client_id: &gatewayClient gateway
-          apiToken: &gatewaySecret 7ed9992425bbdcf03f323310205b2fea34bd9e1a255346f99066800d9440d6f7
       extraEnv:
-         GITLAB_URL: *renku-gitlab-url
-    proxy:
-      secretToken: f89ddee5ba10f2268fcefcd4e353235c255493095cd65addf29ebebf8df86255
+        GITLAB_URL: *renku-gitlab-url
```

Comment:
```suggestion
        GITLAB_URL: *renkuGitlabURL
```

Reply:
oops good catch! this was lost when merging with master